node/src/components/consensus.rs

```diff
@@ -294,7 +294,7 @@ where
             } => handling_es.handle_timer(era_id, timestamp, timer_id),
             Event::Action { era_id, action_id } => handling_es.handle_action(era_id, action_id),
             Event::MessageReceived { sender, msg } => handling_es.handle_message(sender, msg),
-            Event::NewPeer(peer_id) => handling_es.handle_new_peer(peer_id),
+            Event::NewPeer(_peer_id) => Effects::new(),
```

Comment:
Let's just get rid of this event.

Reply:
Removed in [`b3b0e0b` (#1204)](https://github.com/CasperLabs/casper-node/pull/1204/commits/b3b0e0ba5d7c0d59063979af745148202c17baf5)